src/test/java/com/squareup/kotlinpoet/TestFiler.kt

```diff
@@ -53,27 +53,36 @@ internal class TestFiler(
   fun getOriginatingElements(path: Path) = originatingElementsMap[path] ?: throw NullPointerException("Could not find $path")
 
   override fun createSourceFile(
-      name: CharSequence, vararg originatingElements: Element): JavaFileObject {
+    name: CharSequence,
+    vararg originatingElements: Element
+  ): JavaFileObject {
     val relative = name.toString().replace(".", separator) + ".kt" // Assumes well-formed.
     val path = fileSystemRoot.resolve(relative)
     originatingElementsMap[path] = originatingElements.toList()
     return Source(path)
   }
 
-  override fun createClassFile(name: CharSequence, vararg originatingElements: Element)
-      = throw UnsupportedOperationException("Not implemented.")
+  override fun createClassFile(name: CharSequence, vararg originatingElements: Element) =
+      throw UnsupportedOperationException("Not implemented.")
 
-  override fun createResource(location: JavaFileManager.Location, pkg: CharSequence,
-      relativeName: CharSequence, vararg originatingElements: Element): FileObject {
+  override fun createResource(
+    location: JavaFileManager.Location,
+    pkg: CharSequence,
+    relativeName: CharSequence,
+    vararg originatingElements: Element
+  ): FileObject {
     val relative = pkg.toString().replace(".", separator) + separator + relativeName
     val path = fileSystemRoot.resolve(relative)
     originatingElementsMap[path] = originatingElements.toList()
     return Source(path)
   }
 
   override fun getResource(
-      location: JavaFileManager.Location, pkg: CharSequence, relativeName: CharSequence)
-      = throw UnsupportedOperationException("Not implemented.")
+    location: JavaFileManager.Location,
+    pkg: CharSequence,
+    relativeName: CharSequence
+  ) =
+      throw UnsupportedOperationException("Not implemented.")
```

Comment:
nit: merge this with previous line

Reply:
5fca64f772bad7052c1836d3fde8e485210d4395